spec/pb_kits/playbook/kits/textarea_spec.rb

```diff
@@ -7,24 +7,19 @@
 
   it { is_expected.to define_partial }
 
-  it { is_expected.to define_string_prop(:error) }
+  it { is_expected.to define_boolean_prop(:error) }
+  it { is_expected.to define_string_prop(:error_message) }
   it { is_expected.to define_string_prop(:label) }
-  it { is_expected.to define_string_prop(:object) }
-  it { is_expected.to define_string_prop(:method) }
-  it { is_expected.to define_string_prop(:placeholder) }
   it { is_expected.to define_string_prop(:value) }
-  it { is_expected.to define_boolean_prop(:dark)
-                      .with_default(false) }
-  it { is_expected.to define_prop(:rows)
-                      .of_type(Playbook::Props::Number)
-                      .with_default(4) }
+  it { is_expected.to define_boolean_prop(:dark).with_default(false) }
+  it { is_expected.to define_hash_prop(:input_options).with_default({}) }
 
   describe "#classname" do
     it "returns namespaced class name", :aggregate_failures do
       expect(subject.new({}).classname).to eq "pb_textarea_kit resize_none"
-      expect(subject.new({dark: true}).classname).to eq "pb_textarea_kit dark resize_none"
-      expect(subject.new({error: "Something is wrong"}).classname).to eq "pb_textarea_kit error resize_none"
-      expect(subject.new({dark: true, error: "Something is wrong"}).classname).to eq "pb_textarea_kit dark error resize_none"
+      expect(subject.new({dark: true}).classname).to eq "pb_textarea_kit resize_none dark"
```

Comment:
Nit: When only key/value pairs are passed in Ruby, it implicitly makes them a hash. So this is the same thing, and slightly slicker (not a blocker).

```suggestion
      expect(subject.new(dark: true).classname).to eq "pb_textarea_kit resize_none dark"
```